airflow/models.py

```diff
@@ -2485,7 +2485,10 @@ def __lt__(self, other):
     def __hash__(self):
         hash_components = [type(self)]
         for c in self._comps:
-            val = getattr(self, c, None)
+            if c == 'task_ids':
```

Comment:
If only it were so easy :) task_ids is a property already but it returns a list, which is unhashable.